applications/RANSApplication/python_scripts/formulations/fractional_step/fractional_step_k_omega_rans_formulation.py

```diff
@@ -0,0 +1,41 @@
+# import kratos
+import KratosMultiphysics as Kratos
+
+# import formulation interface
+from KratosMultiphysics.RANSApplication.formulations.rans_formulation import RansFormulation
+
+# import formulations
+from KratosMultiphysics.RANSApplication.formulations.incompressible_potential_flow import IncompressiblePotentialFlowRansFormulation
+from KratosMultiphysics.RANSApplication.formulations.turbulence_models.k_omega_rans_formulation import KOmegaRansFormulation
+from KratosMultiphysics.RANSApplication.formulations.fractional_step.fractional_step_velocity_pressure_rans_formulation import FractionalStepVelocityPressureRansFormulation
+
+class FractionalStepKOmegaRansFormulation(RansFormulation):
+    def __init__(self, model_part, settings):
+        super().__init__(model_part, settings)
+
+        default_settings = Kratos.Parameters(r'''
+        {
+            "formulation_name": "fractional_step_k_epsilon",
+            "incompressible_potential_flow_initialization_settings": {},
+            "fractional_step_flow_solver_settings": {},
+            "k_omega_solver_settings": {},
+            "max_iterations": 1
+        }''')
+
+        settings.ValidateAndAssignDefaults(default_settings)
+
+        if (not settings["incompressible_potential_flow_initialization_settings"].IsEquivalentTo(
+                Kratos.Parameters("{}"))):
+            self.incompressible_potential_flow_formulation = IncompressiblePotentialFlowRansFormulation(model_part, settings["incompressible_potential_flow_initialization_settings"])
+            self.AddRansFormulation(self.incompressible_potential_flow_formulation)
```

Comment:
(this is a comment aside of this PR)

I guess you use the potential flow formulation to initialize the velocity and pressure fields. I'm facing the same situation for the compressible NS solver. Considering that this is something useful for any CFD-related problem, it would be nice to standarize it.

Reply:
that would be nice for any CFD simulation. :)